packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -81,6 +81,7 @@ export interface AutocompleteRenderInputParams {
     className: string;
     startAdornment: React.ReactNode;
     endAdornment: React.ReactNode;
+    onMouseDown: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
```

Comment:
Maybe `React.MouseEventHandler` is better?

```suggestion
    onMouseDown: React.MouseEventHandler;
```

Reply:
Fair, `React.MouseEventHandler` is more flexible. updated here https://github.com/mui/material-ui/pull/44183/commits/8e4626475e89552be3dda9f4125694e67e63dc5c